src/Microsoft.DotNet.Build.Tasks/GetNetCoreAppVersionsFromFile.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GetNetCoreAppVersionsFromFile : Task
+    {
+        [Required]
+        public string PathToVersionsFile { get; set; }
+
+        [Required]
```

Comment:
I'll remove the parameter since you are right, it doesn't really make sense and I can just return all.